openid-federation-1_0.xml

```diff
@@ -6733,42 +6736,72 @@ HTTP/1.1 302 Found
 	  as the <spanx style="verb">crit</spanx> value.
 	  Use of this claim is OPTIONAL.
 	</t>
+	<t>
+	  This claim is used in this specification in <xref target="entity-statement"/>
+	  to identify claims not defined by this specification
+	  that MUST be understood and processed
+	  when used in Entity Statements.
+	</t>
       </section>
 
       <section anchor="idClaim" title='"id" (Identifier) Claim'>
 	<t>
 	  The <spanx style="verb">id</spanx> (identifier) claim
```

Comment:
"id" as used by JSON-LD is a string.  So it's not in conflict.

This is half off-topic, but it's odd to me that JSON-LD didn't register any of its claims.  Part of my thinking was that it would be better get the "id" claim registered in a way that matches common usage than to leave it in limbo.  Then JSON-LD's usage would be conforming and could be considered a profile of the registered claim.